### ISSUE TITLE
Term 1, Lesson 4, Section 5

### ISSUE THREAD

**Issue by var7** [udacity/sdc-issue-reports#1111]
Mistake in the last line - "Linear regression will lead to neural networks, which is a much more advanced classification tool."

This should be Logistic Regression instead of Linear Regression. Can confuse students.

**Comment by baumanab** [comment]
Thanks for bringing this up. It does seem like both would work and it certainly does depend on the context. Essentially we can solve both types of regression problems with NNs, the main difference is how we handle network output. For continuous outputs (linear regression) we don't activate the final output. For discrete outputs (logistic regression) we activate the final output via sigmoid (binary classification) or softmax (multi-classification).

**Comment by var7** [comment]
Thanks for explaining that to me. Yes, NNs can solve both types of problems. It did not occur to me. 

In the context of the lesson, since, we're talking about classification and logistic regression I thought it was a typo. Maybe the lesson can be updated to include some part of your explanation?
